designer/client/src/stylesheets/editor.scss

```
@@ -107,6 +107,8 @@ p code {
   }
 }
 
+$box-border: solid 2px #000;
```

Can we keep to `$govuk-*` and `$app-*` suffixes? Cheers @alexluckett 

```suggestion
$app-box-border: solid 2px #000;
```

---

(You'll see the other variables at the top)

---

Done